identity-android/src/main/java/com/android/identity/android/mdoc/transport/L2CAPServer.kt

```diff
@@ -167,7 +162,16 @@ internal class L2CAPServer(val listener: Listener) {
     }
 
     fun sendMessage(data: ByteArray) {
-        writerQueue.add(data)
+        val bsb = ByteStringBuilder()
+        val length = data.size.toUInt()
+        bsb.apply {
```

Comment:
Seems like we should define ByteStringBuilder.append(value: Int) function, otherwise this snippet will proliferate more and more.

Reply:
Yes, think we generally need to have a central place for this. As argued below, think it needs to be done in a separate PR.